middleware/error-handler.js

```diff
@@ -1,10 +1,34 @@
-const { CustomAPIError } = require('../errors')
+// const { CustomAPIError } = require('../errors')
 const { StatusCodes } = require('http-status-codes')
+
 const errorHandlerMiddleware = (err, req, res, next) => {
-  if (err instanceof CustomAPIError) {
-    return res.status(err.statusCode).json({ msg: err.message })
+
+  let customError={
+    //set default
+ statuscode: err.statusCode || StatusCodes.INTERNAL_SERVER_ERROR ,
+ msg: err.mrssgae || 'somthing went wrong . try again later'
```

Comment:
There's a typo so the error message was not being used:
```suggestion
 msg: err.message|| 'something went wrong . try again later'
```